presto-native-execution/presto_cpp/presto_protocol/presto_protocol.h

```diff
@@ -28,7 +28,7 @@
 
 #include <folly/Format.h>
 #include <utility>
-#include "presto_cpp/external/json/json.hpp"
+#include "presto_cpp/external/json/nlohmann/json.hpp"
```

Comment:
@amitkdutta @majetideepak Curious, how was this change made? Was it a manual edit? We should not edit this file manually, but rather generate it.

Reply:
@czentgr Christian, curious, how was this change made? Was it a manual edit? We should not edit this file manually, but rather generate it. It looks like we still need to update presto_cpp/presto_protocol/presto_protocol-json-hpp.mustache file.

Reply:
@mbasmanova Yes, it was a manual edit. You are right, the presto_protocol-json-hpp.mustache is missing the change. The presto-protocol.h is normally generated, but not in every build.
If someone regenerates the presto_protocol files then it will break the build at this moment.

Reply:
@czentgr Thank you for helping fix this. In the future, please, make sure not to edit auto-generated files.

Reply:
@mbasmanova Yes. I didn't notice that it was a generated file.
Do you think I can add a comment after the license header and highlight that this file should not be edited and the README should be consulted?
And do the same for the cpp. This way if someone opens the file it is at the to indicating a generated file?

Reply:
Never mind. It is already there. How am I not seeing these things.

Reply:
@mbasmanova my bad for overlooking this.